dist-material/licenses/license-build.buf.protoc-gen-validate-protoc-gen-validate.txt

```diff
@@ -0,0 +1 @@
+https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
If it is not a special Apache 2.0 license, we actually don't have to copy the text or a single line.

Reply:
This is more like an overkill.  

They use license-eye `-o` to copy the dependencies' license files, and because some dependencies don't have a possible license file in their jar and license-eye has no clue how to find one, license-eye would generate an empty file(or file with the license-eye's internal structure) for these dependencies, to avoid the invalid license files, I suggested to manually copy this file no matter what content it is.

Reply:
It is fine to have these files, just not necessary.
If skywalking-eyes want to keep consistent ways for various licenses, it is fine.